release_notes/topics/4_0_0_final.adoc

```diff
@@ -1,3 +1,15 @@
-= Test Feature
+= Client Scopes and suport for OAuth 2 scope parameter
 
-Some test feature.
\ No newline at end of file
+We added support for Client Scopes. Client Scopes replaced Client Templates. They allow more flexibility and better support for `scope` parameter,
+which we never properly supported until now! In relation to this, there are also some changes on consent screen. The items on consent
+screen are now links to client scopes instead of protocol mappers and roles as it was before. See our documentation and migration guide
+for more details.
+
+= OAuth 2 Certificate Bound Access Tokens
+
+Thanks to community contribution from Takashi Norimatsu, we know partially implement specification
+https://tools.ietf.org/html/draft-ietf-oauth-mtls-08[OAuth 2.0 Mutual TLS Client Authentication and Certificate Bound Access Tokens] .
+More accurately we have support for the Certificate Bound Access Tokens now. If your confidential client is able to use 2-way SSL,
+{project_name} will be able to add the hash of the client certificate into the tokens issued for the client. At this moment,
+it's just  the {project_name} itself, which verifies the token hashes (for example during `refresh token` requests).
+We plan to add support to adapters as well. We also plan to add support for Mutual TLS Client Authentication.
```

Comment:
Add this last as a new paragraph "Thanks to <a href="https://github.com/tnorimat">tnorimat</a> for the contribution."